app/presenters/coronavirus/coronavirus_page_presenter.rb

```diff
@@ -1,4 +1,4 @@
-class CoronavirusPagePresenter
+class Coronavirus::CoronavirusPagePresenter
```

Comment:
Once https://github.com/alphagov/collections-publisher/pull/1265/commits/4a67dd4db2420f425e17c6eb2485914708ff49af is in it might be nicer if this could be named in a similar way so that it's `Coronavirus::PagePresenter`

Reply:
@JonathanHallam Thank you, and yup I've got that in https://github.com/alphagov/collections-publisher/pull/1268 in commit https://github.com/alphagov/collections-publisher/pull/1268/commits/4ec79c5056b52cb0dd2e974b087bbba41b4a5f54

Reply:
nice one 🎉